### PR TITLE
Add bokeh version test

### PR DESCRIPTION
**Pull request by laurenmarietta**
I was inspired (frustrated?) by #425 to write up a quick test that will fail if the Bokeh versions in our HTML templates don't match our environment.

(This will fail Jenkins until #429 is merged.)

### CODE REVIEW

File: jwql/tests/test_plotting.py [hunk 1]
```diff
@@ -36,3 +46,28 @@ def test_bar_chart():
...
+
+    template_paths = os.path.join(JWQL_DIR, 'website/apps/jwql/templates', '*.html')
+    all_web_html_files = glob.glob(template_paths)
+
+    for file in all_web_html_files:
+        with open(file) as f:
+            content = f.read()
+
+        # Find all of the times "bokeh-#.#.#' appears in a template
+        html_versions = re.findall(r'(?<=bokeh-)\d+\.\d+\.\d+', content)
```
Comment (bourque) [inline jwql/tests/test_plotting.py]: @laurenmarietta Would this also include instances of `bokeh-widgets-#.#.#` as well?  I think we have a few of those scattered around our templates.

**Comment by bourque** [comment]
When testing this, I noticed that we have a fair amount of `href`s pointing to `bokeh` amongst our various templates.  I wonder if those are actually necessary, since `bokeh` is imported in `base`, and each template extends `base`.  It might be worth seeing if removing the non-`base` `bokeh` imports actually affects anything.  If not, better to remove them I think.

File: jwql/tests/test_plotting.py (hunk 1 above)
Comment (laurenmarietta) [reply jwql/tests/test_plotting.py]: Good idea!

**Comment by laurenmarietta** [comment]
I think I will try to address the issue of where `bokeh` is getting imported as part of #438, rather than here.

**Comment by bourque** [comment]
The Jenkins builds for this are failing for this (see #443), but since it appears to be an issue unrelated to these code changes, I will go ahead and merge this.

**Pull request merged**